pending/smart_switch_pc_and_kies.xml

```diff
@@ -12,53 +12,79 @@
 
     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
     GNU General Public License for more details.
 
     You should have received a copy of the GNU General Public License
-    along with this program. If not, see <http://www.gnu.org/licenses/>.
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+    @app Samsung Mobiles Programs (Smart Switch PC and Kies)
     @url https://www.samsung.com/us/support/owners/app/smart-switch
-    @cleanerversion v0.1.1
-    @cleanerdate 2019-03-11
+    @os Windows
+    @cleanerversion v0.2.5
+    @cleanerdate 2019-03-14
     @cleanerby https://github.com/Tobias-B-Besemer
     @tested ok 
     @testeddate 
     @testedby 
-    @note 
+    @note Kies was replaced by Smart Switch PC!
+    @note One cleaner for Smart Switch PC and Kies together, because they share files/folders!
+    @note This cleaner is "unstable"! Please improve it before you use it!
 
 -->
 <cleaner id="samsung_mobiles" os="windows">
   <label>Smart Switch PC and Kies</label>
-  <description>Samsung Mobiles - Smart Switch PC and Kies</description>
-  <!--One cleaner for Smart Switch PC and Kies together, because they share files.-->
+  <description>Phone management</description>
   <option id="device_error_recovery_logs">
     <label>Device Error Recovery Logs</label>
-    <description>Delete the Device Error Recovery Logs</description>
-    <action command="delete" search="glob" path="%ProgramData%\Samsung\Device Error Recovery\default-*.log"/>
+    <description>Delete the logs</description>
+    <action command="delete" search="walk.files" path="%ProgramData%\Samsung\Device Error Recovery\default-*.log"/>
```

Comment:
How to use `walk.all`:

* Pass only directories in the path
* Do not pass specific files
* Wildcard pattern is OK

Here it looks like you want `search="glob"` to match these log files

Reply:
It's `search="walk.files"`, and if wildcards works with `search="walk.all"`, it should do with `search="walk.files"`, too, or?